scripts/import.py

```diff
@@ -143,7 +154,15 @@ def main(arguments: argparse.Namespace) -> None:
     else:
         backend = FakeTracBackend()
 
-    add(arguments.base, backend, arguments.year)
+    with contextlib.ExitStack() as stack:
+        if arguments.cache:
+            elements: Optional[Dict[str, int]] = stack.enter_context(
```

Comment:
Should this be a MutableMapping rather than a Dict?

Reply:
I went for `Dict` only as that's what `ElementsCache` returns, though perhaps it ought to be `MutableMapping` (especially since we do need it to be the actual object that the cache yields for it to work as expected.

Reply:
d7fa217